src/tests/java/htsjdk/samtools/util/SamLocusIteratorTest.java

```diff
@@ -93,6 +121,43 @@ public void testEmitUncoveredLoci() {
                 expectedReads = 0;
             }
             Assert.assertEquals(li.getRecordAndPositions().size(), expectedReads);
+            // make sure that we are not accumulating indels
+            Assert.assertEquals(li.getDeletedInRecord().size(), 0);
+            Assert.assertEquals(li.getInsertionInRecord().size(), 0);
+        }
+        Assert.assertEquals(pos, 100001);
+
+    }
+
+    @Test
+    public void testEmitUncoveredLociWithIndels() {
```

Comment:
Instead of copy/pasting the whole method with an additional call to `setIncludeIndels(true)` you could just wrap the body of the original method with `for (final boolean incIndels : new boolean[][ {true, false}) {`